[32mINFO    [0m pytest_operator.plugin:plugin.py:834 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.5.3    unsupported  01:35:17Z

App            Version  Status  Scale  Charm        Channel  Rev  Address         Exposed  Message
config-server           active      1  mongodb-k8s  6/edge    54  10.152.183.203  no       
mongos-k8s              active      1  mongos-k8s              0  10.152.183.195  no       
shard0                  active      1  mongodb-k8s  6/edge    54  10.152.183.104  no       Primary

Unit              Workload  Agent  Address       Ports  Message
config-server/0*  active    idle   10.1.111.141         
mongos-k8s/0*     active    idle   10.1.111.136         
shard0/0*         active    idle   10.1.111.142         Primary

[32mINFO    [0m pytest_operator.plugin:plugin.py:840 Juju error logs:

unit-config-server-0: 01:30:31 ERROR unit.config-server/0.juju-log config-server:4: Failed to add shard shard0 to the config server, error=OperationFailure("Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed', '$clusterTime': {'clusterTime': Timestamp(1727400631, 1), 'signature': {'hash': b'\\x176\\x96\\x83V\\x04\\xf1\\x0c\\xe7C\\xe4\\x9d?5\\xfeMjI\\xe0\\x9f', 'keyId': 7419129075500843012}}, 'operationTime': Timestamp(1727400631, 1)}")
unit-config-server-0: 01:30:31 ERROR unit.config-server/0.juju-log config-server:4: shard0 shard does not have the same auth as the config server.

[32mINFO    [0m pytest_operator.plugin:plugin.py:904 Forgetting main...